apps/src/BriskMatch.cpp

```diff
@@ -0,0 +1,127 @@
+
+#include <opencv2/core.hpp>
+#include <opencv2/features2d.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgcodecs.hpp>
+
+#include <iostream>
+#include <vector>
```

Comment:
Move these includes before the opencv ones. The order in executables should be:

1. System/Std lib includes
2. External library includes
3. Our library includes